README.md

```diff
@@ -76,6 +76,8 @@ information on how to submit your changes to this repository.
 
 ## Sample view
 
+See ./examples for docker-compose file with Grafana if you'd like to quickly get a test environment up and running.
```

Comment:
On 2nd thought, perhaps I'll add the volumes portion, or should I just leave that out of the compose file since this is meant to be an example?